test/CNStakedKLAY.ts

```diff
@@ -374,6 +374,14 @@ describe('CNStakedKLAY', () => {
       });
     });
 
+    describe('#stakeFor()', () => {
+      it('should be able to stake for other account', async () => {
+        await cnStakedKLAY.for.bob.stakeFor(accounts.alice.address, { value: ETHER });
+        await expectBalanceOf(accounts.alice).to.equal(ETHER);
+        await expectBalanceOf(accounts.bob).to.equal(0n);
```

Comment:
nit: this can be rephraed with:
```typescript
await expect(cnStakedKLAY.for.bob.stakeFor(accounts.alice.address, { value: ETHER })).
  to.changeTokenBalances(cnStakedKLAY, [accounts.alice.address, accounts.bob.address], [ETHER, 0n]);
```